src/System.Memory/src/System/Buffers/BuffersExtensions.cs

```diff
@@ -13,6 +13,26 @@ public static class BuffersExtensions
         /// Returns position of first occurrence of item in the <see cref="ReadOnlySequence{T}"/>
         /// </summary>
         public static SequencePosition? PositionOf<T>(this ReadOnlySequence<T> sequence, T value) where T : IEquatable<T>
+        {
+            SequencePosition position = sequence.Start;
+            SequencePosition result = position;
```

Comment:
What is the point of the `position` variable if you immediately reassign it to `result`? Also could getting this value be moved inside the if statement where it is used?

Reply:
Copypasta mistake